sklearn/model_selection/__init__.py

```diff
@@ -0,0 +1,58 @@
+from ._split import BaseCrossValidator
+from ._split import KFold
+from ._split import LabelKFold
+from ._split import StratifiedKFold
+from ._split import LeaveOneLabelOut
+from ._split import LeaveOneOut
+from ._split import LeavePLabelOut
+from ._split import LeavePOut
+from ._split import ShuffleSplit
+from ._split import LabelShuffleSplit
+from ._split import StratifiedShuffleSplit
+from ._split import PredefinedSplit
+from ._split import train_test_split
+from ._split import check_cv
+
+from ._validation import cross_val_score
+from ._validation import cross_val_predict
+from ._validation import learning_curve
+from ._validation import permutation_test_score
+from ._validation import validation_curve
+
+from ._search import GridSearchCV
+from ._search import RandomizedSearchCV
+from ._search import ParameterGrid
+from ._search import ParameterSampler
+from ._search import fit_grid_point
+
+__all__ = ('BaseCrossValidator',
+           'GridSearchCV',
+           'KFold',
+           'LabelKFold',
+           'LabelShuffleSplit',
+           'LeaveOneLabelOut',
+           'LeaveOneOut',
+           'LeavePLabelOut',
+           'LeavePOut',
+           'ParameterGrid',
+           'ParameterSampler',
+           'PredefinedSplit',
+           'RandomizedSearchCV',
+           'ShuffleSplit',
+           'StratifiedKFold',
+           'StratifiedShuffleSplit',
+           'check_cv',
+           'cross_val_predict',
+           'cross_val_score',
+           'fit_grid_point',
+           'learning_curve',
+           'permutation_test_score',
+           'train_test_split',
+           'validation_curve')
+
+
+ALL_CVS = (KFold, LabelKFold, LeaveOneLabelOut, LeaveOneOut, LeavePLabelOut,
+           LeavePOut, ShuffleSplit, LabelShuffleSplit, StratifiedKFold,
+           StratifiedShuffleSplit, PredefinedSplit)
+
+LABEL_CVS = (LabelKFold, LeaveOneLabelOut, LeavePLabelOut, LabelShuffleSplit,)
```

Comment:
Can we have a separate file, `_constants.py` inside `model_selection`? (This will also hold the best param dict that will be added by @zermelozf soon?)

Also @amueller